### PR TITLE
Release v0.4

### PR DESCRIPTION
**Pull request by mr-archano**
## Scope of the PR

We want to release a new version of the plugin on Novoda Bintray maven repo, to ship the changes from #27 and #28.

## Considerations and implementation
- Bumped version for release
- Updated `README`
- Updated `CHANGELOG`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,5 +142,5 @@ Tool | Android | Java | Documentation |
 
 #### Support for sharable configurations
 
-We plan to add support for consuming rules (eg: configuration files for Checkstyle or PMD, default exclude filters, etc) via a
+The plugin can consume rules (eg: configuration files for Checkstyle or PMD, default exclude filters, etc) via a
 separate artifact you can share across projects. _More info to come._
```
Comment (frapontillo) [inline README.md]: Is "more info to come" still valid?

Comment (mr-archano) [inline README.md]: Not sure. I'd like to expand this with an example, but we have no publicly deployed artifact for shared rules yet, hence the `More info to come` still there.

Comment (ataulm) [inline README.md]: Maybe remove it until we have info?

**Comment by ouchadam** [comment]
leaving to @frapontillo

File: README.md (hunk 1 above)
Comment (frapontillo) [inline README.md]: I'm happy to leave it then. It just reads as if the feature isn't implemented, which it is.

Comment (mr-archano) [inline README.md]: Good point guys. I'll get rid of it.

**Pull request merged**